src/app/badges/components/user/user.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-user',
+  templateUrl: './user.component.html',
+  styleUrls: ['./user.component.scss']
+})
+export class UserComponent implements OnInit {
+  user = {userPic: '../assets/user-pic.jpg', userFirstName: 'Sonya', userLastName: 'Podliesna'};
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
```

Comment:
empty constructor and ngOnInit :)
please double check all files and remove the unused ones

Reply:
fixed